conda/recipes/cuproj/meta.yaml

```diff
@@ -74,17 +74,11 @@ requirements:
     {% endif %}
     - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
     - python
-    - rmm ={{ minor_version }}
     - cupy>=12.0.0
 
 test:            # [linux64]
   imports:       # [linux64]
     - cuproj     # [linux64]
-  requires:
-    - cupy>=12.0.0
-    - cuspatial ={{ minor_version }}
-    - rmm ={{ minor_version }}
```

Comment:
`cuproj`'s tests do use `cuspatial`, but those tests aren't run during conda builds. The only test being run here is `import cuproj`, which shouldn't need any additional test-specific dependencies (if it did, that might indicate a missing `run:` dependency!).